sqld/src/rpc/proxy.rs

```diff
@@ -306,7 +306,8 @@ impl Proxy for ProxyService {
         };
 
         tracing::debug!("executing request for {client_id}");
-        let (results, state) = db.execute_program(pgm).await.unwrap();
+        let auth = crate::auth::Authenticated::Authorized(crate::auth::Authorized::FullAccess);
+        let (results, state) = db.execute_program(pgm, auth).await.unwrap();
```

Comment:
this is incorrect. The permissions should be passed by the replica, otherwise, they're useless

Reply:
ah, I tricked myself into thinking this code is only used internally after it already passed validation. Will fix asap